Source/WebCore/platform/graphics/mac/ColorMac.mm

```diff
@@ -151,6 +153,47 @@ Color semanticColorFromNSColor(NSColor *color)
     return cache.get().get(color);
 }
 
+std::pair<Color, Color> lightAndDarkColorsFromNSColor(NSColor *color)
```

Comment:
Maybe this should return a struct instead of an `std::pair` so that it's easy to know which color represents the light color and which one represents the dark one?

Reply:
I think this is made clear by the method name.